src/main/java/edu/greenblitz/tobyDetermined/commands/swerve/MoveToPos.java

```diff
@@ -49,36 +49,40 @@ public void initialize() {
         xController.setGoal(new TrapezoidProfile.State(pos.getX(), 0));
         yController.setGoal(new TrapezoidProfile.State(pos.getY(), 0));
         rotationController.setGoal(new TrapezoidProfile.State(pos.getRotation().getRadians(), 0));
-        
+
     }
-    
+
     public void debugDashboard(double xCalc, double yCalc, double rotationCalc){
         SmartDashboard.putBoolean("at goal x",xController.atGoal());
         SmartDashboard.putBoolean("at goal y",yController.atGoal());
         SmartDashboard.putNumber("x cal",xCalc);
         SmartDashboard.putNumber("y cal",yCalc);
         SmartDashboard.putNumber("rot cal",rotationCalc);
-        
+        SmartDashboard.putNumber("tol", xController.getPositionTolerance());
+
     }
 
     @Override
     public void execute() {
-        double xCalc = xController.atGoal() ? 0 : xController.calculate(SwerveChassis.getInstance().getRobotPose().getX());
-        double yCalc = yController.atGoal() ? 0 : yController.calculate(SwerveChassis.getInstance().getRobotPose().getY());
-        double rotationCalc = rotationController.atGoal() ? 0 : rotationController.calculate(swerve.getChassisAngle());
+        double xCalc = xController.calculate(SwerveChassis.getInstance().getRobotPose().getX());
+        xCalc = xController.atGoal() ? 0 : xCalc;
+        double yCalc = yController.calculate(SwerveChassis.getInstance().getRobotPose().getY());
+        yCalc = yController.atGoal() ? 0 : yCalc;
+        double rotationCalc = rotationController.calculate(SwerveChassis.getInstance().getChassisAngle());
+        rotationCalc = rotationController.atGoal() ? 0 : rotationCalc;
         swerve.moveByChassisSpeeds(
                 xCalc,
                 yCalc,
                 rotationCalc,
                 SwerveChassis.getInstance().getChassisAngle()
         );
         if(debugDash) { this.debugDashboard(xCalc, yCalc, rotationCalc); }
```

Comment:
If you use one liners don't use {}, but even better, don't do it as a one liner.